modules/atomic-widgets/props-resolver/transformers/styles/background-image-overlay-transformer.php

```diff
@@ -18,13 +18,13 @@ public function transform( $value, $key ) {
 
 		$image_url = $this->get_image_url( $value['image-src'], $value['resolution'] ?? self::DEFAULT_RESOLUTION );
 
-		$background_style = "url(\" $image_url \")";
+		$background_style = "url(\"$image_url\")";
 
-		if ( $value['repeat'] ) {
+		if ( ! empty( $value['repeat'] ) ) {
```

Comment:
Do we need the `! empty()` condition? Could you indicate in what case this will make a difference?